src/components/sidenav/sidenav.ts

```diff
@@ -12,6 +12,7 @@ import {
     EventEmitter,
     Renderer
 } from '@angular/core';
+import { Observable } from 'rxjs/Rx';
```

Comment:
I've posted in the old PR as well, but it's still not addressed. 

You should be able to only import the `Observable` here.
```ts
import {Observable} from 'rxjs/Observable';
```